examples/inspection/plot_partial_dependence.py

```diff
@@ -253,7 +253,6 @@
 # Let's make the same partial dependence plot for the 2 features interaction,
 # this time in 3 dimensions.
 import numpy as np
-from mpl_toolkits.mplot3d import Axes3D
```

Comment:
This should be kept as in https://github.com/scikit-learn/scikit-learn/blob/5afd5e160bb731a0445c960fd94740080a44ebd7/examples/cluster/plot_cluster_iris.py#L25-L27
otherwise the documentation fails with older version of matplotlib